frontend/src/lib/Quiver.ts

```diff
@@ -242,7 +246,7 @@ interface DataFrameCell {
   // For "blank" cells "contentType" is undefined.
   // For "columns" cells "contentType" is always set to "unicode"
   // (ArrowJS automatically converts them to strings).
-  contentType?: string
+  contentType?: Type
```

Comment:
Looks like this comment is out of date, since contentType is no longer just a string.